docs/UserGuide.adoc

```diff
@@ -472,21 +472,50 @@ Fees are automatically marked as `private`. `private` details can only
 be seen using the `viewall` command, or for fees specifically, the `viewfees` command.
 ****
 
+[NOTE]
+====
+*NOTE*: You MUST list persons first before using this command! +
+i.e. Indices are selected after first carrying out the any of the `list` commands.
+====
+
```

Comment:
Er. Do we really though? Cause you can also get a list from find etc etc. I think we don't accommodate for testers incapable of basic logic

Reply:
LOL. Ok makes sense, let me remove it